src/api_v2/models/user.rs

```diff
@@ -0,0 +1,73 @@
+use std::time::{SystemTime, UNIX_EPOCH};
```

Comment:
Consider using [`chrono`](https://docs.rs/chrono/latest/chrono/) for more time handle flexibility.

Reply:
no need, just to get unix timestamp